pybossa/view/projects.py

```diff
@@ -3228,8 +3228,6 @@ def process_quiz_mode_request(project):
                 if u['quiz']['config']['enabled']:
                     quiz['status'] = 'in_progress'
                 quiz['config']['enabled'] = u['quiz']['config']['enabled']
-                quiz['config']['completion_mode'] = u['quiz']['config']['completion_mode']
-                quiz['config']['short_circuit'] = (quiz['config']['completion_mode'] == 'short_circuit')
```

Comment:
Just to confirm, it appears that the lines removed were updating the values for `enabled` and `completion_mode` from the form data submitted by the user.

Removing these lines will no longer update those values from the submitted form data. Is that the expected behavior? I only mention this, as the line before it is still updating its value from the submitted form data. Thank you.

Reply:
Yes, that is correct, `completion_mode` and `short_circuit` gets handled on previews line when the project settings gets applied. Basically I only want to update `enabled` on the users based on the data coming from the front end. Thats why I have remove `completion_mode` and `short_circuit`.